applications/teletype/src/teletype_bindings.erl

```diff
@@ -36,11 +36,22 @@ flush_mod(Module) ->
 
 -spec notification(kz_json:object()) -> 'ok'.
 notification(JObj) ->
+    kz_util:put_callid(JObj),
     {EventCategory, EventName} = kz_util:get_event_type(JObj),
     RoutingKey = ?ROUTING_KEY(EventCategory, EventName),
     lager:debug("dispatching notification ~s", [RoutingKey]),
-    _ = kazoo_bindings:map(RoutingKey, JObj, []),
-    'ok'.
+    Res = kazoo_bindings:map(RoutingKey, JObj, []),
+    case kazoo_bindings:succeeded(Res, fun filter_out_failed/1) of
+        [] ->
+            lager:debug("notification ~s did not result in successes", [RoutingKey]),
+            teletype_util:send_update(JObj, <<"failed">>, <<"not_succeeded">>);
```

Comment:
Is there no way to get more info than failed/not_succeeded? Or is that intended?